### PR TITLE
Fetched entity meta value based on lang as well

### PR DESCRIPTION
**Pull request by grafitto**
fixes None

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/api/services/informationextraction/getFiles.ts [hunk 1]
```diff
@@ -78,7 +80,7 @@ async function getFilesForTraining(templates: ObjectIdSchema[], property: string
     { limit: MAX_TRAINING_FILES_NUMBER }
   )) as (FileType & FileEnforcedNotUndefined)[];
 
-  const indextedEntities = objectIndex(entities, e => e.sharedId);
+  const indextedEntities = objectIndex(entities, e => e.sharedId + e.language);
```
Comment (LaszloKecskes) [inline app/api/services/informationextraction/getFiles.ts]: Typo in variable name.

**Review by LaszloKecskes — CHANGES_REQUESTED** [review]
Aside from the typo, looks good.

File: app/api/services/informationextraction/getFiles.ts (hunk 1 above)
Comment (grafitto) [reply app/api/services/informationextraction/getFiles.ts]: Done

**Review by LaszloKecskes — APPROVED** [review]
Looks good.

**Pull request merged**